code/core/src/manager/components/sidebar/SidebarBottom.tsx

```diff
@@ -113,6 +114,8 @@ export const SidebarBottomBase = ({ api, notifications = [], status = {} }: Side
   const hasWarnings = warnings.length > 0;
   const hasErrors = errors.length > 0;
 
+  const bootTestProviders = useCallback(() => api.emit(TESTING_MODULE_BOOT_REQUEST), [api]);
```

Comment:
This event shouldn't be sent on subsequent hovers

Reply:
You're going to have to deal with the event arriving multiple times anyway.

The user might open multiple browsers or refresh, etc.

I'll leave it up to you to decide if adding a check here is worth the additional work/complexity)

Reply:
Why not? Providers may want to shut down after a while. They should already take care not to boot more than once.